tests/query/processors/test_uniq_killswitch.py

```diff
@@ -0,0 +1,65 @@
+from copy import deepcopy
+
+import pytest
+
+from snuba import state
+from snuba.clickhouse.columns import ColumnSet
+from snuba.clickhouse.query import Query
+from snuba.query import SelectedExpression
+from snuba.query.data_source.simple import Table
+from snuba.query.expressions import Column, FunctionCall, Literal
+from snuba.query.processors.uniq_killswitch import UniqKillswitchProcessor
+from snuba.request.request_settings import HTTPRequestSettings
+
+test_data = [
+    (
+        Query(
+            Table("events", ColumnSet([])),
+            selected_columns=[
+                SelectedExpression(
+                    "alias",
+                    FunctionCall("alias", "uniq", (Column(None, None, "column1"),)),
+                ),
+                SelectedExpression(
+                    "alias2",
+                    FunctionCall("alias2", "count", (Column(None, None, "column2"),)),
+                ),
+            ],
+        ),
+        Query(
+            Table("events", ColumnSet([])),
+            selected_columns=[
+                SelectedExpression("alias", Literal("alias", 0),),
+                SelectedExpression(
+                    "alias2",
+                    FunctionCall("alias2", "count", (Column(None, None, "column2"),)),
+                ),
+            ],
+        ),
+    ),
+]
+
+
+@pytest.mark.parametrize("input_query, expected_query", test_data)
```

Comment:
nit: calling this `expected_query` and then having clauses where it's not expected is confusing. maybe call it `killswitched_query` or something